src/main/java/de/ellpeck/rockbottom/init/AbstractGame.java

```diff
@@ -223,7 +222,6 @@ public void init(IGameInstance game, IApiHandler apiHandler, IEventHandler event
     public void postInit(IGameInstance game, IApiHandler apiHandler, IEventHandler eventHandler) {
         TileLayer.init();
         ContentManager.init(this);
-        ConstructionRegistry.postInit();
```

Comment:
This is why construction registry existed. We can't cache recipes just by adding them as final fields because they must be loaded after initialisation.